### PR TITLE
[PLATFORM-1421, PLATFORM-1417] Deprecate API keys functionality, move stream partitions setting to its own section

### PR DESCRIPTION
**Pull request by juhah**
Addressing two issues here at the same time:

* [PLATFORM-1417 Move stream partitions setting to its own section](https://streamr.atlassian.net/browse/PLATFORM-1417) 
* [PLATFORM-1421 Deprecate API keys functionality](https://streamr.atlassian.net/browse/PLATFORM-1421)

Stream edit view:

Screen Shot 2020-07-01 at 13 33 01
Screen Shot 2020-07-01 at 13 33 08

Profile:
Screen Shot 2020-07-01 at 13 33 20

Other changes:
* I changed the profile input widths on desktop to match the designs
* Removed unnecessary components (`PermissionKeyFieldEditor`, `CredentialsControl`, `PermissionCredentialsControl`)
* Replaced separate CSS with `styled-components` in touched components where possible

### CODE REVIEW

File: app/src/shared/components/StatusLabel/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export const LightGrayTheme = {
+    background: '#A3A3A3',
+    color: 'white',
+}
+
+const Label = styled.div`
+    background-color: ${({ theme }) => theme.background};
+    color: ${({ theme }) => theme.color};
```
Comment (fonty1) [inline app/src/shared/components/StatusLabel/index.jsx:17]: pretty cool!

**Comment by juhah** [comment]
> Is the read only Stream page update captured in another ticket? Seems like partitions need to be moved away from the details section there too + lots more.

Looks there was a ticket i missed, [PLATFORM-1420](https://streamr.atlassian.net/browse/PLATFORM-1420), will add that here 👍

**Comment by juhah** [comment]
Added new fields to read-only stream page:

Screen Shot 2020-07-03 at 10 58 07
Screen Shot 2020-07-03 at 10 58 00

**Comment by fonty1** [comment]
> Added new fields to read-only stream page:
> 
> Screen Shot 2020-07-03 at 10 58 07
> Screen Shot 2020-07-03 at 10 58 00

LGTM 👍

**Pull request merged**